smartsim/database/orchestrator.py

```diff
@@ -356,10 +357,13 @@ def is_active(self) -> bool:
         :return: True if database is active, False otherwise
         :rtype: bool
         """
-        if not self._hosts:
+        if not self.hosts:
             return False
```

Comment:
Is the try catch needed here? It looks like the hosts attr (for some reason) can throw this `SSDBFilesNotParseable` error